src/java/htsjdk/variant/vcf/AbstractVCFCodec.java

```diff
@@ -752,9 +755,12 @@ private final static boolean isVCFStream(final InputStream stream, final String
         return new LazyGenotypesContext.LazyData(genotypes, header.getSampleNamesInOrder(), header.getSampleNameToOffset());
     }
 
-    private final String[] INT_DECODE_ARRAY = new String[10000];
+    private final int LARGE_SIZE = 10000;
+    private final String[] INT_DECODE_ARRAY = new String[2*LARGE_SIZE];
```

Comment:
I am not sure how specific I should get with this one. I don't  think there should be any mention of PL. I was thinking about something like "For warning if there is a very large number of genotype fields due to an excessive alternate alleles".